.npmignore

```diff
@@ -0,0 +1,28 @@
+dist/stats.json
```

Comment:
Would it be easier to just ignore everything, and add reverse rules for the stuff you want to keep?